Client/Frontend/Contextual Hint/ContextualHintCopyProvider.swift

```diff
@@ -0,0 +1,103 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+/// This specifies the public facing interface any consumers will interact with.
+protocol ContextualHintCopyProviderProtocol {
+    func getCopyFor(_ copyType: ContextualHintCopyType, of hint: ContextualHintType) -> String
+}
+
+/// `ContextualHintCopyProvider` exists to provide the requested description or action strings back
+/// for the specified `ContextualHintType`.
+struct ContextualHintCopyProvider: ContextualHintCopyProviderProtocol, FeatureFlaggable {
+    typealias CFRStrings = String.ContextualHints
+
+    /// Arrow direction infuences toolbar copy, so it exists here.
+    var arrowDirection: UIPopoverArrowDirection?
```

Comment:
If this isn't used outside this struct (and, given then name of the struct, it shouldn't be used outside of it), then making this private might make intent more clear. My first reaction before looking at the rest of the code was, "this shouldn't be here" because, it's accessible nature led me to believe you were using it in other places.